python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -572,6 +576,30 @@ def __volumeSummary( plug ) :
 
 		],
 
+		"attributes.velocityScale" : [
+
+			"description",
+			"""
+			Scale the vector used in vdb  motion blur computation.
+			""",
+
+			"layout:section", "Volume",
+			"label", "Velocity Scale",
+
+		],
+
+		"attributes.velocityFPS" : [
+
+			"description",
+			"""
+			Set the frame rate used in vdb motion blur computation.
```

Comment:
Set**s**, and VDB.

Do we have any more information about this attribute? When should one use this vs the scale for instance, and how does it work?

Reply:
I've sent an email to the solid angle support to get a little bit more information about this parameter. Ideally they'd sent us the formula used to compute the final vector for blurring... Will report back.

Reply:
Solidangle has told me now that they do this to determine if motion blur needs to be computed at all:

```
#define AI_EPSILON         (1.0e-4f)
if ( velocity_scale / velocity_fps < AI_EPSILON ) then motion blur is disabled.
```

I'm just guessing here, but I think they're assuming those motion vectors to be in `<your favorite spatial unit> / seconds` and the fps parameter is a scale to convert to frames, while the other parameter is to scale the spatial unit or to lookdev the blur? Does that make sense?